src/views/virtualmachines/details/components/details/hooks/useSSHService.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+
+import { ServiceModel } from '@kubevirt-ui/kubevirt-api/console';
+import { V1VirtualMachineInstance } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import { K8sResourceCommon, useK8sWatchResource } from '@openshift-console/dynamic-plugin-sdk';
+
+type UseSSHServiceProps = (vmi: V1VirtualMachineInstance) => {
```

Comment:
I don't think this should be named *Props since it's defining the return type instead of the inputs.